import/party_scraper/src/party_scraper/pipelines.py

```diff
@@ -29,10 +29,16 @@
 class JsonWithEncodingPipeline(object):
     def _post_elastic(self, content):
 
+        # generate unique id for event
+        eventId = 'pa_scr_' + content['title'] + content['start_date']
+        content['id'] = eventId
+
+        # insert into elasticsearch
         Elasticsearch(elasticsearch_url, http_auth=http_auth, scheme=scheme).index(
             index=(f"{index_prefix}events"),
             doc_type="_doc",
-            body=content
+            body=content,
+            id=eventId
```

Comment:
gibt das einen Grund, warum wir z.B. den location_name nicht in die elasticsearch mitnehmen? Wenn ich z.B. einfach nur nach Events in der Sputnikhalle suchen will?

Reply:
Hm, wie kommst Du da drauf? location_name geben wir mit soweit ich weiß.